src/vs/editor/contrib/hover/browser/contentHover.ts

```diff
@@ -730,15 +726,12 @@ export class ContentHoverWidget extends ResizableContentWidget {
 		if (!this._visibleData) {
 			return;
 		}
```

Comment:
Unsure about removing this. Looks like this was the only reader of `stoleFocus` and we either remove it alltogether or validate when/why is used

Reply:
Indeed, I placed it back. The variable stoleFocus now also checks the value of focusedHoverKey.